src/styles/Footer.scss

```diff
@@ -0,0 +1,53 @@
+@import "./constants.scss";
+
+.footer {
+    position: absolute;
+    bottom: 0px;
+    left: 0px;
+    background-color: $primaryDark;
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    width: 100vw;
+    height: 80px;
+
+    .footer-logo-container {
+        max-width: 1024px;
+        width: 85%;
+        display: flex;
+        justify-content: space-between;
+        flex-direction: row;
+
+        a {
+            padding: 1%;
+
+            img {
+                height: 100%;
+                width: auto;
+            }
+        }
+    }
+}
+
+@media only screen and (max-width: 426px) {
+    .footer {
+        position: absolute;
```

Comment:
do we need all these styles if they are specified for the non media version as well?